src/ssl/ensemble.py

```diff
@@ -53,14 +69,15 @@ def drop_ensemble(self):
         """
         self.ensemble = []
 
-    def fit_ensembĺe(self, instances, classes):
+    def fit_ensemble(self, instances, classes):
         """
         Treina os classificadores presentes no cômite
         """
+
         for classifier in self.ensemble:
             self.fit_single_classifier(classifier, instances, classes)
 
-    def fit_single_classifier(self, classifier: FlexConC, instances, classes):
+    def fit_single_classifier(self, classifier, instances, classes):
```

Comment:
```suggestion
    def fit_single_classifier(self, classifier, instances, classes) -> ???:
```

tipos das variáveis.

Reply:
feito